fallzahlen_kanton_total_csv_v2/COVID19_Fallzahlen_Kanton_AG_total.csv

```diff
@@ -57,4 +57,6 @@ date,time,abbreviation_canton_and_fl,ncumul_tested,ncumul_conf,new_hosp,current_
 2020-05-06,14:45,AG,,1141,,24,5,5,960,36,https://www.ag.ch/media/kanton_aargau/themen_1/coronavirus_1/lagebulletins/200508_KFS_Coronavirus_Lagebulletin_50.pdf
 2020-05-07,14:45,AG,,1145,,19,4,4,970,37,https://www.ag.ch/media/kanton_aargau/themen_1/coronavirus_1/lagebulletins/200508_KFS_Coronavirus_Lagebulletin_50.pdf
 2020-05-08,14:45,AG,,1149,,16,3,3,990,37,https://www.ag.ch/media/kanton_aargau/themen_1/coronavirus_1/lagebulletins/200508_KFS_Coronavirus_Lagebulletin_50.pdf
+2020-05-09,,AG,,1152,,,,,,,https://www.ag.ch/media/kanton_aargau/themen_1/coronavirus_1/lagebulletins/200511_KFS_Coronavirus_Lagebulletin_51.pdf
```

Comment:
`1152` should be changed to `1153`: "Am Samstag, 09.05.2020 wurden 4 Neuansteckungen gemeldet"